pkg/registry/relocator.go

```diff
@@ -10,7 +10,7 @@ import (
 
 	"github.com/google/go-containerregistry/pkg/authn"
 	"github.com/google/go-containerregistry/pkg/name"
-	"github.com/google/go-containerregistry/pkg/v1"
+	v1 "github.com/google/go-containerregistry/pkg/v1"
```

Comment:
I feel like I am missing some idiomatic Go stuff. would love to learn more (to avoid this)

Reply:
this was just from running `goimports` so I just left it

Reply:
cool.. I will stop undoing it :)